test/functionalTest/cases/2015_notification_templates/notification_templates_cache_refresh.test

```diff
@@ -130,13 +130,13 @@ Date: REGEX(.*)
 ============================================
 PATCH http://127.0.0.1:REGEX(\d+)/notify?a1=a1&a3=13&a2=a2&type=Thing&id=E1
 Fiware-Servicepath: /
-Ngsiv2-Attrsformat: custom
```

Comment:
Based on last travis check, the changes in this and the other .test file are needed to make ft to work. Some headers change their order.

This is a bit weird and I'd like to confirm is ok. However, it goes beyond my knowledgue, as it seems related with some internal in Flask or Werkzeug). I have asked in SOF https://stackoverflow.com/questions/55734118/flask-werkzeug-change-headers-order-after-upgrade. Please upvote ;)

Reply:
Based in the "answer" of that question and this one https://stackoverflow.com/questions/55751419/getting-request-headers-in-the-same-order-they-are-received-in-flask/55752367#55752367, is fine for me *in the context of this PR*.

*For the future* I have created an issue to think about it: https://github.com/telefonicaid/fiware-orion/issues/3473

Reply:
Any idea why these two tests got changes in HTTP headers order and no other tests?

Reply:
Yes. In the SOF answers and links to duplicated questions.

Reply:
NTC